test/wrappers/vroom_test.rb

```diff
@@ -756,4 +756,17 @@ def test_mixed_shipments_and_services
     assert_equal 0, result[:unassigned].size
     assert_equal 5, result[:routes][0][:activities].size
   end
+
+  def test_correct_route_collection
+    problem = VRP.lat_lon_two_vehicles
+    problem[:services].each{ |service|
+      service[:skills] = ['s1']
+    }
+    problem[:vehicles].last[:skills] = [['s1']]
+
+    result = OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:vroom] }}, TestHelper.create(problem), nil)
+    assert result
+    assert_equal 2, result[:routes].size
+    assert_equal problem[:services].size + 2, result[:routes].find{ |route| route[:vehicle_id] == problem[:vehicles].last[:id] }[:activities].size
```

Comment:
```suggestion
    assert_equal problem[:services].size, result[:routes].find{ |route| route[:vehicle_id] == problem[:vehicles].last[:id] }[:activities].count{ |activity| activity[:service_id }
```

Reply:
Unless we want to make sure that the depots are there or not?

Reply:
If so, I suggest we make it more explicit.

Reply:
Figuring out if depots are there or not is not the purpose of this test, so I will follow the recommendations of @fonsecadeline